### DISCUSSION TITLE
error HTTP response: 401, after update to 2.16.4

### DISCUSSION THREAD

**Discussion by devops-cora** [terrakube-io/terrakube#551] [Q&A] [answered]
After update to 2.16.4 my terraform remote execution  only return: 

Error: Failed to upload configuration files: error HTTP response: 401

There something that I need to look?

**Comment by alfespa17** [comment, marked as answer]
Hello 

You can get that error if you are using terraform 1.6.X there is an open issue about that

https://github.com/AzBuilder/terrakube/issues/532

I still need to check what changed in that terraform version when uploading the configuration files.
